src/app/dev/DevToys.Tools/Tools/Converters/Date/DateConverterCommandLineTool.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System.Globalization;
+using DevToys.Tools.Helpers;
+using DevToys.Tools.Models;
+using Microsoft.Extensions.Logging;
+
+namespace DevToys.Tools.Tools.Converters.Date;
+
+[Export(typeof(ICommandLineTool))]
+[Name("DateConverter")]
+[CommandName(
+    Name = "dateConverter",
+    ResourceManagerBaseName = "DevToys.Tools.Tools.Converters.Date.DateConverter",
+    DescriptionResourceName = nameof(DateConverter.Description))]
+internal sealed class DateConverterCommandLineTool : ICommandLineTool
+{
+    [CommandLineOption(
+        Name = "number",
+        Alias = "n",
+        DescriptionResourceName = nameof(DateConverter.NumberOptionDescription))]
+    internal long? NumberOption { get; set; }
+
+    [CommandLineOption(
+        Name = "date",
+        Alias = "d",
+        DescriptionResourceName = nameof(DateConverter.DateOptionDescription))]
+    internal string? DateOption { get; set; }
+
+    [CommandLineOption(
+        Name = "epoch",
+        Alias = "e",
+        DescriptionResourceName = nameof(DateConverter.EpochOptionDescription))]
+
+    internal string? EpochOption { get; set; }
+
+    [CommandLineOption(
+        Name = "timezone",
+        Alias = "tz",
+        IsRequired = true,
```

Comment:
Could we make this entry optional and detect the local timezone in `InvokeAsync` when `TimeZoneOption` is null?

Also, to understand how to use this parameter, I had to look at the code then read the documentation of `TimeZoneInfo.FindSystemTimeZoneById` to find what I should type to get it to work. This is quite difficult from a user standpoint.

The following code should help with getting the list of available entries:
```
            foreach (TimeZoneInfo z in TimeZoneInfo.GetSystemTimeZones())
            {
                Console.WriteLine(z.Id);
            }
```

I think this list may be localized and change based on the user language, which is another challenge.

Unfortunately, the `CommandLineOption` attribute doesn't allow us to inject some dynamically generated documentation, therefore, here are my suggestions:
1. Run the code above in a Console app and copy the result. Paste it in `DateConverter.TimezoneOptionDescription` as part of the explanation on how to use this parameter. In the comment of `DateConverter.TimezoneOptionDescription`, explain how you got this list. It will be important for translators that they run this code in order to get an accurate list.
2. Change `CommandLineOption` to support dynamically-generated documentation. This is likely a lot of work to do and I seriously discourage you to spend a lot of time on it.
3. If my ideas above aren't good enough or require too much more work, then let's create a GitHub issue for us to track, and please add a `// TODO` comment with a link to that issue.

Reply:
Yeah for the moment I added the ms doc link for timezone: https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/default-time-zones?view=windows-11&WT.mc_id=DT-MVP-5001664&source=post_page-----cff0e2b37f52--------------------------------#time-zones

But we nee to find a way to handle these case.